src/WindowManager.vala

```diff
@@ -367,6 +343,8 @@ namespace Gala {
 
                 plugin_manager.load_waiting_plugins ();
 
+                daemon_manager.start ();
```

Comment:
Any reason it's here and not in `start`? I think it would make sense to create DaemonManager and start it in a single function.

Reply:
I'm not sure I think it's a remnant from an earlier version I had but I changed it to start in construct and it still works 🤷